deployment/systemd/downloader.service

```diff
@@ -2,7 +2,7 @@
 Description=YouTube Downloader
 
 [Service]
-WorkingDirectory=/opt/radiodan/rde/downloader
+WorkingDirectory=/opt/radiodan/rde/services/downloader
```

Comment:
This was `rde` because initially it was called "RadioDan Evolution". I know, I know, sorry.